packages/core/src/profile/client/patchUserAttribute.js

```diff
@@ -0,0 +1,28 @@
+import client, { adaptError } from '../../helpers/client';
+import join from 'proper-url-join';
+
+/**
+ * Method responsible for updating specific user attribute.
+ *
+ * @function patchUserAttribute
+ * @memberof module:profile/client
+ *
+ * @param {number} userId - User's id to be filtered for.
+ * @param {string} attributeId - The attribute id to be filtered for.
+ * @param {object} data - User attribute data.
```

Comment:
You need to specify which properties are in this `data` param instead of just `object`. i.e.
`@param {UserAttibuteData} data - User attribute data.` and then specify that ` UserAttibuteData`